okhttp/src/main/kotlin/okhttp3/ConnectionPool.kt

```diff
@@ -152,5 +161,28 @@ class ConnectionPool internal constructor(
     @JvmField val backoffDelayMillis: Long = 60 * 1000,
     /** How much jitter to introduce in connection retry backoff delays */
     @JvmField val backoffJitterMillis: Int = 100,
+    /**
+     * The maximum number of concurrent calls per connection for a given address.
+     * Set this value to 1 to disable HTTP/2 connection coalescing
+     */
+    @JvmField val maximumConcurrentCallsPerConnection: Int? = null,
   )
+
+  /**
+   * A default policy for all connections. Overridable by [AddressPolicy]
```

Comment:
AddressPolicy is internal, so this effectively makes a public feature expose an internal one.

I'd like @swankjesse take on this.

I guess I wonder whether this indicates rather than two tiered policy (ConnectionPoolPolicy then AddressPolicy), that a single ConnectionPoolPolicy interface that can answer either by global or per address logic.